test/acceptance/user_announcement_test.exs

```diff
@@ -6,9 +6,9 @@ defmodule ConstableWeb.UserAnnouncementTest do
 
     session
     |> visit(announcement_path(Endpoint, :new, as: user.id))
-    |> fill_in("announcement_title", with: "Hello World")
+    |> fill_in(announcement_title(), with: "Hello World")
```

Comment:
I would prefer to see this reference the field by its label. I noticed though that the form does not have labels. We should add some.

Reply:
This might be better off as another pull request though.

Reply:
Yeah, I was noticing that finding elements by css seems to be the way all of the acceptance tests are written.